src/widgets/topicFrequencyGraph.js

```diff
@@ -0,0 +1,91 @@
+import { Op } from 'sequelize';
+import {
+  ActivityReport,
+} from '../models';
+import { REPORT_STATUSES } from '../constants';
+
+// copied from /frontend/src/pages/ActivityReports/constants.js
+export const topics = [
+  'Behavioral / Mental Health',
```

Comment:
Another reason to refactor and use the Topics table in the database (in a different PR :))- we wouldn't have to rename here as well. This was the reason I didn't see the correct number of topics on the bar graph.